Content.Shared/SS220/IgnoreLightVision/Components/AddIgnoreVisionOverlayComponent.cs

```diff
@@ -2,19 +2,25 @@
 
 using Robust.Shared.Serialization;
 
-namespace Content.Shared.SS220.IgnoreLightVision;
+namespace Content.Shared.SS220.IgnoreLightVision.Components;
 
 public abstract partial class AddIgnoreLightVisionOverlayComponent : Component
 {
     [DataField]
     public IgnoreLightVisionOverlayState State = IgnoreLightVisionOverlayState.Off;
+
     [DataField]
     public float VisionRadius = 8f;
+
     [DataField]
     public float HighSensitiveVisionRadius = 2f;
+
     [DataField]
     public bool AddAction = false;
 
+    [DataField]
+    public bool IsIconsAlwaysSeen;
```

Comment:
```suggestion
    public bool ChangeIconOverlaySpace;
```

Ща найду пасту... Не нашёл.

Короче bool название с Is... это маркер плохо названия, зачастую Is можно убрать без потери смысла.

Наверное такое будет лучше